src/jobs/job.rs

```diff
@@ -99,11 +100,14 @@ pub struct Job {
     #[serde(default)]
     pub on_success: Option<Action>,
 
+    /// Whether to display how many files triggered the current run.
     pub show_changes_count: Option<bool>,
 
+    /// Color overrides applied to the UI while the job runs.
     #[serde(default)]
     pub skin: BaconSkin,
 
+    /// Per-job sound settings that override the global configuration.
```

Comment:
Job fields are flattened into the global config too, so you can't say it's "per-job".

Reply:
```suggestion
    /// Notification sounds
```